src/chainparams.cpp

```diff
@@ -242,7 +242,7 @@ class CTestNetParams : public CChainParams {
 
         fMiningRequiresPeers = true;
         fDefaultConsistencyChecks = false;
-        fRequireStandard = false;
+        fRequireStandard = true;
```

Comment:
fRequireStandard should stay as "false" on testnet.

Reply:
Only after HF. Not for current testing